### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by RodinGolodin**

### CODE REVIEW

**Review by thienNH0 — DISMISSED** [review]

**Review by thienNH0 — DISMISSED** [review]

File: Assets/Plugins/README.md [hunk 1]
```diff
@@ -1,105 +1,101 @@
...
 
-`Android` : Settings in Unity as described in the [Customize platform-specific settings](https://docs.skymavis.com/mavis/mavis-id/guides/integrate-unity#customize-platform-specific-settings) section in the integration guide.
+* An app created in the [Developer Console](https://developers.skymavis.com/console/applications/).
+* Permission to use Mavis ID. Request in **Developer Console > your app > App Permission > Sky Mavis Account (OAuth 2.0) > Request Access**.
+* A client ID that you can find in **Developer Console > Products > ID Service > Client ID**.
+* A redirect URI registered in **Developer Console > Products > ID Service > Redirect URI**.
 
-### Initialization
+* To deploy to Android, [Android API level 24](https://developer.android.com/about/versions/nougat) or later.
+* To deploy to iOS, [iOS 12.0](https://developer.apple.com/ios/) or later.
```
Comment (thienNH0) [inline Assets/Plugins/README.md:34]: We upgraded iOS so min version from now 13.0

File: Assets/Plugins/README.md [hunk 2]
```diff
@@ -1,105 +1,101 @@
...
-            }
-        }
-#else
-        SM.MavisId.Init(AppId, DeeplinkSchema);
-#endif
-    }
+void Start()
+{
+    // Client ID registered in the Mavis ID settings in the Developer Console
+    string appId = "cbabcb00-9c99-404b-a6e4-c76b3b59f0d8";
```
Comment (thienNH0) [inline Assets/Plugins/README.md:51]: I don't know that does Sec team allow us public client id although it's just a test project

File: Assets/Plugins/README.md [hunk 3]
```diff
@@ -108,41 +104,135 @@ public async void OnSendTransactionClicked()
...
+## Utilities
+
+The SM.ID.Utils namespace provides utility functions for easier wallet interactions. The following are some common tasks.
+
+### Encode function data
+
+Encodes function data for smart contract interactions.
+
+```csharp
+string EncodeFunctionData(string ABI, object values)
```
Comment (thienNH0) [inline Assets/Plugins/README.md:177]: Please note that, this is readable ABI, that mean the function signature of contract as example , not the json ABI.

Comment (RodinGolodin) [reply Assets/Plugins/README.md:177]: Can you suggest the right way to phrase it?

File: Assets/Plugins/README.md (hunk 2 above)
Comment (RodinGolodin) [reply Assets/Plugins/README.md:51]: We can hide it in env variables, what would the syntax be like?

File: Assets/Plugins/README.md (hunk 1 above)
Comment (RodinGolodin) [reply Assets/Plugins/README.md:34]: Will update!

File: Assets/Plugins/README.md (hunk 2 above)
Comment (thienNH0) [reply Assets/Plugins/README.md:51]: // Client ID registered with Mavis ID
string appID = ${YOUR_CLIENT_ID}

This is the popular that I use

File: Assets/Plugins/README.md (hunk 3 above)
Comment (thienNH0) [reply Assets/Plugins/README.md:177]: It's smart contract function like : function approve(address _spender, uint256 _value) which you can see clearly in smart contract 
For example : https://github.com/OpenZeppelin/openzeppelin-contracts/blob/c304b6710b4b5fcf2a319ad28c36c49df6caef14/contracts/token/ERC20/ERC20.sol#L127

**Review by thienNH0 — APPROVED** [review]

**Pull request merged**